spec/support/shared_examples/abilities/below_agency_with_refund_examples.rb

```diff
@@ -4,4 +4,8 @@
   it "should not be able to view revoked reasons" do
     should_not be_able_to(:view_revoked_reasons, WasteCarriersEngine::RenewingRegistration)
   end
+
+  it "should not be able to refund a payment" do
```

Comment:
This example doesn't cover the finance user types.

Reply:
I am not sure I get the structure of those tests yet 😅 Added more